corehq/apps/domain/management/commands/make_superuser.py

```diff
@@ -20,6 +20,12 @@ def add_arguments(self, parser):
             'username',
         )
 
+        parser.add_argument(
+            '--allow_superuser_management',
```

Comment:
I think you want this to be `allow_superuser_management`, which I think will make the command line argument `--allow-superuser-management` (with `-`s which I think is more standard).

Reply:
adding arguments with the "--" in the code is the way [to make that argument optional ](https://docs.djangoproject.com/en/4.1/howto/custom-management-commands/#accepting-optional-arguments) which I think is what's needed here.

Reply:
I'm sorry my comment here was sloppy, and had a mistake (which you pointed out) that distracted from the main thing I was trying to say. I was just saying that the convention I'm used to is named command line arguments to use hyphens (`-`) instead of underscores (`_`) inside their names, so `--allow-superuser-management`.

Reply:
Thinking a little more about it, I think that it would actually be fine to just not have this flag available, and instead _always_ set `can_assign_superuser` on a superuser that is created through this management command. That's actually what I was suggesting here https://github.com/dimagi/commcare-hq/pull/32027#issuecomment-1224872413, but that might have not come through.

The reason I say it's okay is that those docs for self-hosters bootstrapping a setup is the _only_ time this management command is used—we don't use it in production at all, and if you try it fails. (This was to force the use of proper channels for modifying superuser access https://github.com/dimagi/commcare-hq/pull/30107.) So if someone's using this management command, then they're always going to want the resultant user to be able to have `can_assign_superuser`.

Reply:
Just double checking but using the command in production is still possible for some people, right? At least one user needs to be given the can_assign_superuser privilege after deploy. Or I guess it could be manually set using django manage - is that something we'd want to do?